### PR TITLE
Megaphone cooldown text feedback

### PR DESCRIPTION
**Pull request by MaxIsJoe**
stop bugging me about megaphones they're not broken or unreliable they're on god damn cooldown.

Also if they're emagged now there is no cooldown for them so they're extra annoying now, you asked for this.

CL: [Improvement] added text feedback to notify that a megaphone is on cooldown.
CL: [New] emagging a megaphone removes it's cooldown.

### CODE REVIEW

File: UnityProject/Assets/Scripts/Items/Bureaucracy/Megaphone.cs [hunk 1]
```diff
@@ -34,7 +34,14 @@ private IEnumerator Cooldown()
...
 		public bool RunChecks()
 		{
-			return isOnCooldown == false && isOn == true && pickupable.ItemSlot != null;
+			return IsOnCooldown() && isOn == true && pickupable.ItemSlot != null;
+		}
+
+		private bool IsOnCooldown()
+		{
+			if (isEmmaged) return false;
+			if(isOnCooldown && pickupable.ItemSlot != null && pickupable.ItemSlot.Player != null) Chat.AddExamineMsg(pickupable.ItemSlot.Player.gameObject,"The megaphone is on cooldown..");
```
Comment (NoooneyDude) [inline UnityProject/Assets/Scripts/Items/Bureaucracy/Megaphone.cs]: Could we rephrase this to be more "diegetic"? How about,
```suggestion
			if (isOnCooldown && pickupable.ItemSlot != null && pickupable.ItemSlot.Player != null)
			{
				Chat.AddExamineMsg(pickupable.ItemSlot.Player.gameObject, "The megaphone's internal capacitor is still recharging...");
			}
```
?

File: UnityProject/Assets/Scripts/Items/Bureaucracy/Megaphone.cs [hunk 2]
```diff
@@ -34,7 +34,14 @@ private IEnumerator Cooldown()
 
 		public bool RunChecks()
 		{
-			return isOnCooldown == false && isOn == true && pickupable.ItemSlot != null;
+			return IsOnCooldown() && isOn == true && pickupable.ItemSlot != null;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Items/Bureaucracy/Megaphone.cs]: You're missing == false here

File: UnityProject/Assets/Scripts/Items/Bureaucracy/Megaphone.cs [hunk 3]
```diff
@@ -34,7 +34,14 @@ private IEnumerator Cooldown()
...
+		}
+
+		private bool IsOnCooldown()
+		{
+			if (isEmmaged) return false;
```
Comment (NoooneyDude) [inline UnityProject/Assets/Scripts/Items/Bureaucracy/Megaphone.cs]: Oh, `isEmmaged` should be `isEmagged`.

**Comment by MaxIsJoe** [comment]
done, review pls

**Review by NoooneyDude — APPROVED** [review]
